homeassistant/components/plugwise/config_flow.py

```diff
@@ -29,6 +33,7 @@ def _base_schema(discovery_info):
         base_schema[vol.Required(CONF_HOST)] = str
 
     base_schema[vol.Required(CONF_PASSWORD)] = str
+    base_schema[vol.Optional(CONF_PORT, default=DEFAULT_PORT)] = int
```

Comment:
If it's discovered, the port is not needed from the user right?
If so this can be moved to under L33

Reply:
Good call ... committed (with catchup hence the force)

Reply:
+ collateral 👍